scripts/build.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const del = require('del');
+const rollup = require('rollup');
+const babel = require('rollup-plugin-babel');
+const eslint = require('rollup-plugin-eslint');
+const uglify = require('rollup-plugin-uglify');
+const pkg = require('../package.json');
+
+const bundles = [
```

Comment:
we'll at least need to use babel to transpile from esm imports/exports to commonjs require/exports